Tests/scripts/run_tests.sh

```diff
@@ -19,35 +43,58 @@ CONF_PATH="./Tests/conf.json"
 echo "export GOOGLE_APPLICATION_CREDENTIALS=$GCS_ARTIFACTS_KEY" >> "${BASH_ENV}"
```

Comment:
Why isn't it the same name?
(rhetorical question. I know it's not you...)

Reply:
i don't know